moveit_setup_assistant/moveit_setup_framework/src/urdf_config.cpp

```diff
@@ -97,20 +97,18 @@ void URDFConfig::setPackageName()
   {
     urdf_pkg_name_ = "";
     urdf_pkg_relative_path_ = urdf_path_;  // just the absolute path
+    throw std::runtime_error("URDF/COLLADA file package not found: " + urdf_path_.string());
   }
-  else
+  // Check that ROS can find the package
+  const std::filesystem::path robot_desc_pkg_path = getSharePath(urdf_pkg_name_);
+
+  if (robot_desc_pkg_path.empty())
   {
-    // Check that ROS can find the package
-    const std::filesystem::path robot_desc_pkg_path = getSharePath(urdf_pkg_name_);
-
-    if (robot_desc_pkg_path.empty())
-    {
-      RCLCPP_WARN(*logger_,
-                  "Package Not Found In ROS Workspace. ROS was unable to find the package name '%s'"
-                  " within the ROS workspace. This may cause issues later.",
-                  urdf_pkg_name_.c_str());
-    }
-  }
+    RCLCPP_WARN(*logger_,
+                "Package Not Found In ROS Workspace. ROS was unable to find the package name '%s'"
+                " within the ROS workspace. This may cause issues later.",
```

Comment:
(I know this is from before, but...) This message seems redundant. Maybe just take off that first sentence?

Reply:
I would argue to keep it consistent (with your other comments) that this should stay as the "subject line" as it were for the error that would go into a dialogue box if desired ever. I'm also fine just stripping away the redundant parts for all these error messages instead if so desired.